src/core/core.scale.js

```diff
@@ -1205,28 +1205,55 @@ var Scale = Element.extend({
 		var scaleLabelFont = helpers.options._parseFont(scaleLabel);
 		var scaleLabelPadding = helpers.options.toPadding(scaleLabel.padding);
 		var halfLineHeight = scaleLabelFont.lineHeight / 2;
+		var scaleLabelAlign = scaleLabel.align;
 		var position = options.position;
 		var rotation = 0;
-		var scaleLabelX, scaleLabelY;
+		var scaleLabelX, scaleLabelY, textAlign;
+		var isReverse = me.options.ticks.reverse;
```

Comment:
nit: can you switch this line with the one above it? we usually put lines that initialize variables above the ones that are just declaring variables

Reply:
Done